docs/usage/getting-started/installing-onboarding.md

```diff
@@ -67,6 +67,10 @@ Once you have enabled Renovate on a repository, you will get a "Configure Renova
 
 ![Onboarding PR from Renovate](../assets/images/onboarding.png){ loading=lazy }
 
+<!-- prettier-ignore -->
+!!! note
+    If you self-host Renovate, and want to add the rebase/retry checkbox to Renovate's onboarding PRs: enable the [`onboardingRebaseCheckbox` config option](https://docs.renovatebot.com/self-hosted-configuration/#onboardingrebasecheckbox) first.
```

Comment:
I think we want a relative link here? Right now we're linking to the docs domain (`https://docs.renovatebot.com`).

I always mess up the links, so I'll let a maintainer suggest the correct link. 🙃